app/app/guide/page.tsx

```
@@ -0,0 +1,280 @@
+      {children}
+    </section>
+  );
+}
+
+export default function GuidePage() {
+  return (
+    <main className="mx-auto max-w-[900px] px-6 py-12 space-y-16">
+      {/* Hero */}
+      <div className="text-center space-y-3">
+        <div className="text-[10px] font-medium uppercase tracking-[0.3em] text-[var(--accent)]/50">
+          // documentation
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)